package.json

```diff
@@ -263,7 +263,7 @@
     "@material-ui/core": "^4.12.4",
     "@metamask/abi-utils": "^3.0.0",
     "@metamask/account-api": "^0.9.0",
-    "@metamask/account-tree-controller": "^0.8.0",
+    "@metamask/account-tree-controller": "^0.13.0",
```

Comment:
For reviewers: Other peer deps (`keyring-controller` + `accounts-controller`) will have to be updated at some point, but we need to make a big release for every of them.

For the moment this change should be compatible with the existing deps, we should not have any invalid behaviors with this change.